### PR TITLE
clickhouse IfNull added

### PR DESCRIPTION
**Pull request by d-ganchar**

### CODE REVIEW

File: pypika/clickhouse/nullable_arg.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from pypika import Field
+from pypika.terms import Function
+
+
+class IfNull(Function):
+    def __init__(self, field, alt, alias: str = None, schema: str = None):
```
Comment (twheys) [inline pypika/clickhouse/nullable_arg.py]: if you change this to 
```
 def __init__(self, *args, **kwargs):
     super().__init__('NULLIF', name, alias=alias)
```

then you wouldn't need to write a custom implementation for `get_sql`

Comment (twheys) [reply pypika/clickhouse/nullable_arg.py]: if you want to enforce the args, you could do it like
```
 def __init__(self, term, alt, **kwargs):
    super().__init__('NULLIF', term, alt, **kwargs)
```

**Comment by d-ganchar** [comment]
> no need for custom `get_sql` implementation, as the function syntax is standard.

Thanks. I'll fix it.

File: pypika/tests/clickhouse/test_search_string.py [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-            (Like, "like(toString(\"name\"),'twheys')"),
-            (NotLike, "notLike(toString(\"name\"),'twheys')"),
-        )
 
-        for cls, expected in search_types:
-            self.assertEqual(
-                cls(Field('name'), 'twheys'),
-                expected
-            )
+    @parameterized.expand([
```
Comment (twheys) [inline pypika/tests/clickhouse/test_search_string.py]: btw this library is a really good addition for the types of tests we write in pypika. Appreciate you adding it.

**Comment by twheys** [comment]
Code looks good, just need to fix the build for py3.5

**Comment by twheys** [comment]
nevermind, looks like one of the tests I added is flapping. Going to merge.

**Comment by d-ganchar** [comment]
> nevermind, looks like one of the tests I added is flapping. Going to merge.

https://github.com/kayak/pypika/pull/314#issue-312707452

**Pull request merged**